utils/rpms/daos.spec

```
@@ -248,7 +247,7 @@ install -m 644 utils/systemd/%{server_svc_name} %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{agent_svc_name} %{?buildroot}/%{_unitdir}
 mkdir -p %{?buildroot}/%{conf_dir}/certs/clients
 mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
-install -m src/client/java %{_prefix}/lib/daos/TESTING/ftest/java/
+install -m src/client/java %{_prefix}/lib/daos/TESTING/java/
```

`install -m ` should be followed by a _mode_.  I think you want the `DEST` to be the dir you want the trailing dir component of the `SOURCE` path:

```suggestion
install -m 755 src/client/java %{_prefix}/lib/daos/TESTING/
```

I think.  :-)